test/ussd_public.test.js

```diff
@@ -271,7 +288,7 @@ describe("Hello Mama app", function() {
                 });
             });
 
-            describe("Change states flows - baby messages", function() {
+            describe.skip("Change states flows - baby messages", function() {
```

Comment:
`.skip`s was left in on purpose to facilitate breaking up into different issues.